product-detail-edit/index.php

```diff
@@ -12,7 +12,7 @@
       <div class="panel panel-margin-bottom">
         <div class="panel-body">
           <div class="row">
-            <div class="col-sm-6  col-xs-12">
+            <div class="col-md-6  col-xs-12">
```

Comment:
@j2trumpet are you sure you want to switch from Col-6 to 12 so Early? What about a Col-Sm-8 in between?